File: src/ApproximatorTester/ApproximatorTester.Tests/test/data/Js/ForWithIf.js
{caret}function main(cond)
{
    var query = "select";
    for (var i = 0; i < 10 && i > 0 || query.IsNormalized(); i++)
    {
        var localQ = "from ";
        if (cond)
        {
            localQ += "Table1";
        }
        else
        {
            localQ += "Table2";
        }
        query += localQ;
    }
    execScript(query);
}

File: src/SELApproximator/SELApproximator.Tests/test/data/CSharpFsa/ForWithIf.cs
{caret}namespace N
{
    class Program
    {
        static void Main(string[] args)
        {
            string query = "a";
            for (int i = 0; i < 10 && i > 0 || query.IsNormalized(); i++)
            {
                string localQ = "b";
                if (args.Length != 0)
                {
                    localQ += "c";
                }
                else
                {
                    localQ += "d";
                }
                query += localQ;
            }
            Program.ExecuteImmediate(query);
        }

        static void ExecuteImmediate(string query)
        {
            
        }
    }
}